lib/cli.go

```diff
@@ -23,22 +23,35 @@ import (
 	"os"
 	"sort"
 
-	"github.com/purpleidea/mgmt/bindata"
 	"github.com/purpleidea/mgmt/gapi"
 	_ "github.com/purpleidea/mgmt/lang" // import so the GAPI registers
 	_ "github.com/purpleidea/mgmt/langpuppet"
 	_ "github.com/purpleidea/mgmt/puppet"
 	_ "github.com/purpleidea/mgmt/yamlgraph"
 
-	"github.com/urfave/cli"
+	"github.com/urfave/cli/v2"
 )
 
+// CLIArgs is a struct of values that we pass to the main CLI function.
+type CLIArgs struct {
```

Comment:
I noticed that we use `Cli` in `gapi`: https://github.com/purpleidea/mgmt/blob/master/gapi/gapi.go#L60
Should we change one or the other to be consistent?